addons/website/static/tests/tours/snippet_image_crop.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module */
+
+import wTourUtils from "@website/js/tours/tour_utils";
+
+wTourUtils.registerWebsitePreviewTour('snippet_image_crop', {
+    test: true,
+    url: '/',
+    edition: true,
+}, () => [
+    wTourUtils.dragNDrop({
+        id: 's_text_image',
+        name: 'Text - Image',
+    }),
+    {
+        content: "Select image",
+        trigger: 'iframe .s_text_image img',
+    },
+    {
+        content: "Replace image",
+        trigger: 'iframe .s_text_image img',
+        run: 'dblclick',
+    },
+    {
+        content: "Search for the illustration images",
+        trigger: ".o_select_media_dialog .o_we_search",
+        run: "text city",
+    },
+    {
+        content: "Select the first illustration image",
+        trigger: ".o_we_existing_attachments a.o_we_media_author:contains('Undraw')"
+    },
+    {
+        content: "Click on the first illustration image",
+        trigger: ".o_select_media_dialog .o_we_attachment_highlight",
+    },
+    {
+        content: "Select the image",
+        trigger: 'iframe .s_text_image img',
+        ischeck: true,
+    },
+    {
+        content: "Try to crop the image",
+        trigger: "#oe_snippets .o_we_customize_panel .o_we_user_value_widget[data-crop='true']",
+    },
+    ...wTourUtils.clickOnSave(),
```

Comment:
Check edition mode is closed after save, add extra check step or make sure clickOnSave gets failed without our solution.

Reply:
@msh-odoo In `clickOnSave()` method itself checks that so we don't explicitly need to check it again.